mutmut/__main__.py

```diff
@@ -957,36 +957,59 @@ def should_ignore_for_mutation(self, path):
         return False
 
 
-@lru_cache()
-def read_config():
+def config_reader():
```

Comment:
Doesn't this mean it ONLY reads from pyproject.toml and now ignores setup.cfg? That's no good imo. If mutmut should force pyproject.toml it should at the very least error out if there is a mutmut section in setup.cfg.

Reply:
no, the config reader is your old `s()` function. It returns the old `setup.cfg` reader if there is no pyproject.toml with an `[tool.mutmut]` key.

Reply:
Cool. I cherry-picked this fix into main

Reply:
you can also cherry-pick my fix for the debug option.

Reply:
Done